### DISCUSSION TITLE
No consumption/efficiency in drives dashboard

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#3013] [Q&A]
As you can see in the attached screenshot, some of my drives have no consumption and efficiency values.
Do you know why?
![Screenshot_20221207_195245_Samsung Internet](https://user-images.githubusercontent.com/402122/206271212-f8e35be7-126b-4906-88b7-3779620d5fc6.jpg)

**Comment by cwanja** [comment]
For the shorter drives with the snowflake icon, TeslaMate cannot accurately compute those values. So they remain blank. If you hover over the snowflake icon, you can get a tooltip that will explain. The longer drives can more accurately compute consumption. 

See #2255

**Reply by djpalmis** [reply]
Thank you for your (continuous) precious support!
